test/uk/gov/hmrc/gatekeepercomposeemailfrontend/connectors/UploadDocumentsConnectorSpec.scala

```diff
@@ -65,23 +66,21 @@ class UploadDocumentsConnectorSpec extends AsyncHmrcSpec with BeforeAndAfterEach
   }
   val composEmailServieStub = new ComposeEmailServiceStub
 
-  trait Setup {
-    val httpClient = app.injector.instanceOf[HttpClient]
-
+  trait Setup extends HttpClientV2Support {
     implicit val hc: HeaderCarrier    = HeaderCarrier()
     implicit val appConfig: AppConfig = mock[AppConfig]
     val CREATED                       = 201
     val OK                            = 200
 
-    class UploadDocumentsConnectorSuccess extends UploadDocumentsConnector(httpClient, composEmailServieStub) {
+    class UploadDocumentsConnectorSuccess extends UploadDocumentsConnector(httpClientV2, composEmailServieStub) {
```

Comment:
Looks like someone's keyboard battery was about to expire here. Could we please correct `composEmailServieStub` while you're in here?